### PR TITLE
Remove search button

### LINKED ISSUE #1472: Remove the search button in the archive page
**Is your feature request related to a problem? Please describe.**
- We don't need the search button because the search happens as the user types

**Describe the solution you'd like**
- Remove the search button on the archive page
- https://sefglobal.org/scholarx/archive/

Screenshot 2022-12-11 at 9 15 20 AM

**Are you working on it?**
- No

### PR DESCRIPTION
**Pull request by dilankavishka**
## Purpose

The purpose of this PR is to fix #1472

## Goals

Remove the search button on ScholarX archive page.

## Approach

Removed the search button on ScholarX archive page.

### Screenshots

  
Screenshot 2022-12-16 at 13 27 20

### Preview Link

https://pr-1477-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/archive/index.html [hunk 1]
```diff
@@ -136,10 +136,6 @@ <h6>UNIVERSITY</h6>
           </div>
       </div>
         <div class="col-9">
-            <div class="search-filter">
-                <input id="search" type="text" class="form-control rounded search-bar" placeholder="Search" aria-label="Search" aria-describedby="search-addon" />
-                <button type="button" class="btn btn-outline-primary search-btn">search</button>
```
Comment (anjula-sack) [inline scholarx/archive/index.html:141]: We need to remove only the button @dilankavishka. Keep the search input

**Review by anjula-sack — DISMISSED** [review]

File: scholarx/archive/index.html (hunk 1 above)
Comment (dilankavishka) [reply scholarx/archive/index.html:141]: Done! @anjula-sack

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Can you remove this css also @dilankavishka 

https://github.com/sef-global/sef-site/blob/f437bd654c5751fa6f9128b5118768bf1db0c252/scholarx/archive/styles.css#L10-L12

**Comment by dilankavishka** [comment]
Done! @YoshithaRathnayake

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You! @dilankavishka

**Pull request merged**